dist/api_err.py

```diff
@@ -132,12 +132,61 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
         setting.'''),
 ]
 
+# To ensure our sub-level error returns do not conflict with any other
+# package or the error returns, we use the range -32,000 to -32,199.
+#
+# These numbers cannot change without breaking backward compatibility,
+# and are listed in error value order.
 sub_errors = [
     Error('WT_NONE', -32000,
         'last API call was successful', '''
         This is the default sub-level error code that should be used when there is no
         sub-level error to pair with an error. It indicates that no further context
         exists or is necessary.'''),
+    Error('WT_COMPACTION_ALREADY_RUNNING', -32001,
+        "cannot reconfigure background compaction while it's already running", '''
```

Comment:
@jeremythorp The long descriptions here might not always be accurate. Currently, we lack detailed descriptions for `sub_level_error_code`. Should we discuss and define these error code descriptions and update our tech design doc?

Reply:
hi @zunyiliu , I agree - we should review the error code descriptions and update the tech design doc. 
My suggestion would be to accept the (effectively draft) wording in this PR so that we can merge it and start using these new sub level error codes. Then we add a note in the tech design that the messages need to be reviewed and updated, and also create a ticket (in the project) to revise the text as required once agreement is reached. How does that sound?

Reply:
Sounds good. @jiechenbo I created a ticket [WT-13927](https://jira.mongodb.org/browse/WT-13927) for this, can you schedule that into one of the milestones?